mono/profiler/proflog.h

```diff
@@ -5,7 +5,7 @@
 #define LOG_HEADER_ID 0x4D505A01
 #define LOG_VERSION_MAJOR 0
 #define LOG_VERSION_MINOR 4
-#define LOG_DATA_VERSION 10
+#define LOG_DATA_VERSION 12
```

Comment:
We should change this to single version bump.

Given data version 11 will never be in any released mono, it has no value on its own.